src/deadline_worker_agent/capabilities.py

```diff
@@ -116,6 +116,18 @@ def _get_gpu_memory(*, verbose: bool = True) -> int:
         output_bytes = subprocess.check_output(
             ["nvidia-smi", "--query-gpu=memory.total", "--format=csv,noheader"]
         )
+        output = output_bytes.decode().strip()
+
+        mem_per_gpu: list[int] = []
+        for line in output.splitlines():
+            mem_mib = int(line.replace("MiB", ""))
+            mem_per_gpu.append(mem_mib)
+
+        min_memory = min(mem_per_gpu)
+
+        if verbose:
+            _logger.info("Minimum total memory of all GPUs: %s", min_memory)
+        return min_memory
```

Comment:
Because we have the loop inside the `try`/`except` block, a single GPU with `[N/A]` will cause this function to report 0 when in actuality there may be a GPU available with memory.

I believe we'd want to instead report the minimum memory of all available GPU.